jcli/src/jcli_lib/transaction/mod.rs

```diff
@@ -62,6 +63,8 @@ pub enum Transaction {
     Auth(auth::Auth),
     /// get the message format out of a sealed transaction
     ToMessage(common::CommonTransaction),
+    /// send  a transaction from faucet (simplified method)
+    Send(simplified::SimplifiedTransaction),
```

Comment:
also, a command named `Send` may be a bit confusing, at least to me it suggests network interaction

Reply:
Makes sense